src/txmempool.cpp

```diff
@@ -751,6 +745,10 @@ void CTxMemPool::check(CChainState& active_chainstate) const
         assert(it->GetSizeWithAncestors() == nSizeCheck);
         assert(it->GetSigOpCostWithAncestors() == nSigOpCheck);
         assert(it->GetModFeesWithAncestors() == nFeesCheck);
+        // Sanity check: we are walking in ascending ancestor count order. The non-contextual
+        // check() will have ensured that this count is accurate.
```

Comment:
I don't understand what "non-contextual check()" is referring to here.

Reply:
Leftover from previous iteration of the PR - removed now